label_studio/ml_models/models.py

```diff
@@ -54,11 +57,19 @@ class Meta:
 
     parent_model = models.ForeignKey(ModelInterface, related_name='model_versions', on_delete=models.CASCADE)
 
+    prompt = models.TextField(_('prompt'), null=False, blank=False, help_text='Prompt to execute')
+
     @property
     def full_title(self):
         return f'{self.parent_model.title}__{self.title}'
 
-    prompt = models.TextField(_('prompt'), null=False, blank=False, help_text='Prompt to execute')
+    def delete(self, *args, **kwargs):
+        """
+        Deletes Predictions associated with ModelVersion
+        """
+        model_run = ModelRun.objects.get(model_version=self.id)
```

Comment:
wouldn't this throw an error if there are 2 model runs for this model version, one on GT and one on all tasks? Or did we make that impossible?

Reply:
No, this is for one model version object. 
For separate subsets, there will be a different model version object